docker-compose.yml

```diff
@@ -1,98 +1,233 @@
 version: "3.8"
 
-x-environment:
-  &oncall-environment
-  DATABASE_TYPE: sqlite3
-  BROKER_TYPE: redis
-  BASE_URL: $DOMAIN
-  SECRET_KEY: $SECRET_KEY
-  REDIS_URI: redis://redis:6379/0
-  DJANGO_SETTINGS_MODULE: settings.hobby
-  CELERY_WORKER_QUEUE: "default,critical,long,slack,telegram,webhook,retry,celery"
-  CELERY_WORKER_CONCURRENCY: "1"
-  CELERY_WORKER_MAX_TASKS_PER_CHILD: "100"
-  CELERY_WORKER_SHUTDOWN_INTERVAL: "65m"
-  CELERY_WORKER_BEAT_ENABLED: "True"
+x-oncall-build: &oncall-build-args
+  context: ./engine
+  target: dev
+
+x-env-files: &oncall-env-files
+  - .env.dev
+  - .env.${DB}.dev
+  - .env
+
+x-env-vars: &oncall-env-vars
+  BROKER_TYPE: ${BROKER_TYPE}
 
 services:
-  engine:
-    image: grafana/oncall
+  oncall_ui:
+    container_name: oncall_ui
+    build:
+      context: ./grafana-plugin
+      dockerfile: Dockerfile.dev
+    volumes:
+      - ./grafana-plugin:/etc/app
+      - /etc/app/node_modules
+    profiles:
+      - oncall_ui
+
+  oncall_engine:
+    container_name: oncall_engine
+    build: *oncall-build-args
     restart: always
+    command: "python manage.py runserver 0.0.0.0:8080"
+    env_file: *oncall-env-files
+    environment: *oncall-env-vars
     ports:
       - "8080:8080"
-    command: >
-      sh -c "uwsgi --ini uwsgi.ini"
-    environment: *oncall-environment
     volumes:
-      - oncall_data:/var/lib/oncall
```

Comment:
we cannot get rid of docker-compose.yaml and[docker-compose-mysql-rabbitmq.yml completely because they are used for hobby installations.
Hobby installation can be done without cloning git repo. We need to keep named volumes for such installation. I propose to have two different versions of docker-compose files: for hobby deployments (from docker hub) and for development (build from local context)

I want to keep hobby yaml as simple as possible as it will increase conversion to installation